### PR TITLE
Justified content of the hero section  and afix responsiveness

### LINKED ISSUE #234: Fix Responsiveness of the Landing page

### PR DESCRIPTION
**Pull request by shubhagarwal1**
fix: #234
**Desktop**
Screenshot 2024-10-24 at 12 30 07 AM

**Mobile** 
Screenshot 2024-10-24 at 12 29 47 AM

### CODE REVIEW

**Comment by MitulSonagara** [comment]
image

@shubhagarwal1

File: src/app/(app)/page.tsx [hunk 1]
```diff
@@ -214,3 +207,4 @@ const Page = async () => {
 };
 
 export default Page;
+c
```
Comment (MitulSonagara) [inline src/app/(app)/page.tsx]: fix this c

Comment (shubhagarwal1) [reply src/app/(app)/page.tsx]: > fix this c

fixing it right now

**Comment by shubhagarwal1** [comment]
Hey @MitulSonagara i have fixed the code, please review

**Comment by MitulSonagara** [comment]
@shubhagarwal1 

this is looking too left 
image

**Comment by shubhagarwal1** [comment]
> @shubhagarwal1
> 
> this is looking too left image

should i align it to the right most ???

**Comment by MitulSonagara** [comment]
@shubhagarwal1 

Not right most... align div to center but text to left

**Pull request closed without merging**